modules/web/client-model/src/main/scala/observe/ui/model/ObsSummary.scala

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package observe.ui.model
+
+import cats.Eq
+import cats.Order.given
+import cats.derived.*
+import cats.syntax.all.*
+import io.circe.Decoder
+import io.circe.generic.semiauto.*
+import lucuma.core.model.ConstraintSet
+import lucuma.core.model.ObsAttachment
+import lucuma.core.model.PosAngleConstraint
+import lucuma.core.model.TimingWindow
+import lucuma.core.util.Timestamp
+import lucuma.schemas.decoders.given
+import lucuma.schemas.model.BasicConfiguration
+import lucuma.schemas.model.ObservingMode
+import monocle.Focus
+import org.typelevel.cats.time.*
+
+import java.time.Instant
+import scala.collection.immutable.SortedSet
+
+case class ObsSummary(
```

Comment:
I wonder if this can be shared with explore, we have a very similar class over there

Reply:
That was my initial hunch but after checking, it turns out we request way less information here.